packages/im-ott/src/routes/AppRoutes.js

```diff
@@ -29,6 +31,8 @@ const AppRoutes = () => {
         <Route exact path="/" component={LandingPage} />
         <Route exact path="/watchlist" component={WatchList} />
         <Route exact path="/film/:filmId" component={FilmDetails} />
+        <Route exact path="/player" component={Player} />
```

Comment:
Do we need a separate path for player and seeker ?

Reply:
to test, should i remove all test links?

Reply:
It's okay to keep it for now. Just wanted to know the reason